stdlib/public/core/Range.swift

```diff
@@ -907,3 +907,15 @@ extension Range {
 @available(*, deprecated, renamed: "Range")
 public typealias CountableRange<Bound: Strideable> = Range<Bound>
   where Bound.Stride : SignedInteger
+
+@available(*, deprecated: 4.2, renamed: "PartialRangeFrom")
+public typealias CountablePartialRangeFrom<Bound: Strideable> = PartialRangeFrom<Bound>
+  where Bound.Stride : SignedInteger
+
+@available(*, deprecated: 4.2, renamed: "PartialRangeUpTo")
+public typealias CountablePartialRangeUpTo<Bound: Strideable> = PartialRangeUpTo<Bound>
+  where Bound.Stride : SignedInteger
```

Comment:
I don't think these two ever existed… It was just `5...` that was countable, right?

Reply:
i.e. Just `CountablePartialRangeFrom` needs to be an alias.

Reply:
ohhhh right.... yeah, because that is the only one that can be a sequence cos it has a starting point.

That makes me slightly less embarrassed to have forgotten to do this.